client/src/components/search/searchBar/search-bar-optimized.tsx

```diff
@@ -56,14 +62,19 @@ const SearchBarOptimized = ({
               value={value}
               ref={inputElementRef}
             />
-            <button className='ais-SearchBox-submit' type='submit'>
+            <button
+              className='ais-SearchBox-submit'
+              type='submit'
+              aria-label={t('search.submit')}
+            >
               <Magnifier />
             </button>
             {value && (
               <button
                 className='ais-SearchBox-reset'
                 onClick={onClick}
                 type='button'
+                aria-label={t('search.reset')}
```

Comment:
Same issue here, we don't need to add aria-label. The InputReset already contains the sr-only text "Clear search terms". No need to override this with aria-label on the button.